core-services/prow/02_config/_plugins.yaml

```diff
@@ -372,6 +372,10 @@ approve:
   - openshift-priv/special-resource-operator
   - openshift/special-resource-operator
   require_self_approval: false
+- commandHelpLink: ""
+  repos:
+  - openshift/cluster-cloud-controller-manager-operator
+  require_self_approval: false
```

Comment:
I think we should be requiring self approval here, otherwise when I push a PR, as I'm in the owners, it will get an automatic approve, this is not what we normally do

Reply:
ack, I'll update this part

Reply:
ack, I'll update this part